### PR TITLE
Fix tiny typo "is is" with "is" in index.md

### PR DESCRIPTION
**Pull request by elgalu**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -76,4 +76,4 @@ like-minded individuals, we have a number of open channels for communication.
 
 * To contribute code or documentation changes: please submit a [pull request on Github](https://github.com/docker/machine/pulls).
 
-For more information and resources, please visit the [Getting Help project page](https://docs.docker.com/project/get-help/).
```
Comment (aluzzardi) [inline docs/index.md]: What changes here?

Comment (elgalu) [inline docs/index.md]: looks like a git bug, I didn't change a thing there :/

**Comment by vieux** [comment]
LGTM

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**